### ISSUE TITLE
Manifestation Chapter - D.8. Internet Manifestation type - enhance text and add in some VOD examples

### ISSUE THREAD

**Issue by natashafairbairn** [FIAF/fiaf-cataloguing-manual#85]
Some draft text already in a Word document - just need to copy and paste it in and get some VOD examples

**Comment by heftberger** [comment]
Would that go here or rather in the Appendix for digital examples? In any case, probably easy to do for you if you have text already drafted.

**Comment by heftberger** [comment]
Natasha wrote: "Will double check that one, I can’t remember whether I added it into a branch elsewhere or not, but will root around and find out. I know I hadn’t added in examples."

**Comment by heftberger** [comment]
@natashafairbairn any luck with your search? I think it is important to establish whether you had actually already done this so we don't have this issue sitting and waiting :-) Thanks a lot!

**Comment by natashafairbairn** [comment]
Yes, text is in the main Develop draft already, re. Internet Manifestation type and also in Manifestation/Event Boundaries section. I do want to write up a bit about cataloguing acquisitions from TikTok, Youtube, etc. internet channels (either in a new section somewhere/or else incorporated within existing sections), but will create a new Issue for that when ready to do it. So this issue can be closed and a new issue created for adding VOD examples (I have the ones I want to use it is just a question of formatting and adding in - may need to do in Word and get Paul's assistance in transferring it into Github document.
